src/Plugin/EntityReferenceSelection/CivicrmContactSelection.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\civicrm_entity\Plugin\EntityReferenceSelection;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Entity\Plugin\EntityReferenceSelection\DefaultSelection;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\taxonomy\Entity\Vocabulary;
+
+/**
+ * Provides specific access control for the civicrm_contact entity type.
+ *
+ * This is because `display_name` does not work on LIKE queries, but the
+ * `=` condition is treated as one.
```

Comment:
Is this behavior only for display_name? If so, we should probably check the name of the field being matched and swap LIKE for '='. Otherwise, this could cause problems if the user chooses a different field to autocomplete on.

Reply:
Ah, ok, understood! Thanks :-)